addons/medical/stringtable.xml

```diff
@@ -2215,6 +2215,7 @@
         </Key>
         <Key ID="STR_ACE_Medical_healHitPointAfterAdvBandage">
             <English>Heal fully bandaged hitpoints</English>
+            <Russian>Исцелять полностью перебинтованные части тела</Russian>
```

Comment:
The English string is a little ambiguous here, but this was translated as if it read:
> Restore bandaged body parts to full health (/HP)

Is that the meaning you meant to convey? In other words, in the original sentence, does "hitpoints" stand for "wounds" relative to specific body parts?